src/components/SourceTabs.js

```diff
@@ -229,7 +229,7 @@ const SourceTabs = React.createClass({
         key: source.get("id"),
         onClick: () => selectSource(source.get("id")),
         onContextMenu: (e) => this.onTabContextMenu(e, source.get("id")),
-        title: source.get("url")
+        title: getRawSourceURL(source.get("url"))
```

Comment:
I suppose we could simplify all of this by adding `.replace(/:formatted$/, "")` here, but the function will come in handy in the future.